src/app/cube/shared/featured/featured.component.ts

```diff
@@ -38,7 +39,10 @@ export class FeaturedComponent implements OnInit {
         }
       });
     } else {
-      this.fetchLearningObjects();
+      this.featureService.getFeaturedObjects();
+      this.featureService.featuredObjects.subscribe(objects => {
```

Comment:
This should be unsubscribed when the component is destroyed right?

Reply:
Yes, good catch.